manual/core/performance/README.md

```diff
@@ -0,0 +1,351 @@
+## Performance
+
+This page is intended as a checklist for everything related to driver performance. Most of the
+information is already available in other sections of the manual, but it's linked here for
+easy reference if you're benchmarking your application or diagnosing performance issues.
+ 
+
+### Statements
+
+[Statements](../statements/) are some of the driver types you'll use the most. Every request needs
+one -- even `session.execute(String)` creates a `SimpleStatement` under the hood.
+
+#### Immutability and builders
+
+Statements are immutable: every call to a setter method creates an intermediary copy. If you have
```

Comment:
There is a slight confusion between interface and implementation: it's not the interfaces, but the _default implementations_ thereof that are immutable . Suggestion: "Statements are by default implemented with immutable classes: every call..."